overlay/backend/Makefile

```
@@ -25,7 +25,10 @@ $(TEST_TARGETS):
 .PHONY: pkg-test
 pkg-test:
 	@$(MAKE) -C pkg/
-	
+
+.PHONY: integration-test
+integration-test:
```

Sorry for being picky, but could we change the target to `test-integration` for the sake of consistency. If we created a directory `backend/services/integration` that would create a conflicting target.
```suggestion
.PHONY: test-integration
test-integration:
```